addons/web/static/src/views/list/list_renderer.js

```diff
@@ -879,8 +879,11 @@ export class ListRenderer extends Component {
 
     getOptionalActiveFields() {
         this.optionalActiveFields = {};
-        const optionalActiveFields = browser.localStorage.getItem(this.keyOptionalFields);
+        let optionalActiveFields = browser.localStorage.getItem(this.keyOptionalFields);
         if (optionalActiveFields) {
+            if (typeof optionalActiveFields === "string") {
```

Comment:
it is always a string isn't it? The if seems unnecessary

Reply:
yes, unless we are in makeRAMLocalStorage where we have the object directly (on safari incognito)

but with the other suggestion (modifying `makeRAMLocalStorage`) this would not be necessary :+1: